runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -55,6 +59,8 @@
 @Slf4j
 public class AddressSpaceView extends AbstractView {
 
+    private static final int COUNTER_INIT_VALUE = 0;
```

Comment:
Just inline this? it doesn't seem to be used outside the  `AddressSpaceView.`

Reply:
done.